src/BloomBrowserUI/bookEdit/js/readerTools.ts

```diff
@@ -42,10 +53,82 @@ function processDLRMessage(event: MessageEvent): void {
       model.setMarkupType(parseInt(params[1]));
       return;
 
+    case 'Qtips': // request from accordion to add qtips to marked-up spans
+      // We could make separate messages for these...
+      markDecodableStatus();
+      markLeveledStatus();
+
+      return;
+
     default:
   }
 }
 
+function markDecodableStatus(): void {
+  // q-tips; mark sight words and non-decodable words
+  var editableElements = $(".bloom-content1");
+  editableElements.find('span.' + (<textMarkup>$).cssSightWord()).each(function() {
+    (<qtipInterface>$(this)).qtip({ content: 'Sight word' });
+  });
+
+  editableElements.find('span.' + (<textMarkup>$).cssWordNotFound()).each(function() {
+    (<qtipInterface>$(this)).qtip({ content: 'This word is not decodable in this stage.' });
```

Comment:
Function markDecodableStatus above seems to do the same thing. Do we need both?
Do we need localization here?